tox.ini

```diff
@@ -49,7 +60,8 @@ sitepackages = true
 
 [testenv:lint]
 deps =
-    -rlint-requirements.txt
+    flake8>=3.6.0,<4
+    yamllint>=1.11.1,<2
```

Comment:
The `lint` and `flake8` deps are already in the `testenv.deps` list, so one set of them is redundant and should be removed either from `testenv.deps` or `testenv:lint.deps`, as appropriate.

Reply:
I thought so as well, but it turned out that these are also runtime deps of Molecule and are used not only for linting of the Molecule itself but also in scenarios and tests of those scenarios. We can clean up this mess later in further refactoring. I just want to focus on finishing with the packaging stuff.